code/game/gamemodes/clock_cult/clock_items/judicial_visor.dm

```diff
@@ -169,7 +169,7 @@
 		var/datum/status_effect/belligerent/B = C.apply_status_effect(STATUS_EFFECT_BELLIGERENT)
 		if(!QDELETED(B))
 			B.duration = world.time + 30
-			C.Knockdown(5) //knocks down for half a second if affected
+			C.Knockdown(80) //knocks down for 8 seconds if affected
```

Comment:
Make this like 27 and make the below stun, the Knockdown(15), like 90.
At 27 only people in the outer edge will be able to dodge the explosion, and people in the center won't be able to at all.

Reply:
Ugh I misread pretty much every aspect of this, im cutting the initial stun because it never even worked at (5) and I dont think there should be an initial stun at all, I did increase the belligerent duration for a few seconds though.

I made the actual ending stun 80, for anyone listening the ending stun was 1.5 seconds, not 0.5.

Reply:
if you remove the initial stun it'll be even weaker than it was before; the belligerent duration is the time it takes to explode

Reply:
I've literally never seen the initial stun stun people. I don't think stuns under a second even work.